presto-docs/src/main/sphinx/release/release-306.rst

```diff
@@ -0,0 +1,76 @@
+===========
+Release 306
+===========
+
+General Changes
+---------------
+
+* Fix planning failure for queries containing a ``LIMIT`` after a global
+  aggregation. (:issue:`437`)
+* Fix missing column types in EXPLAIN output. (:issue:`328`)
+* Fix updating peak revocable memory reservation. (:issue:`413`)
+* Fix blocking on memory when revocable memory is used. (:issue:`413`)
+* Fix double memory accounting for spilling aggregations. (:issue:`413`)
+* Fix possible CPU active waiting when window operator spill is triggered. (:issue:`468`)
```

Comment:
I think I know what "possible CPU active waiting" means but this needs to be phrased better. Maybe

> Fix excessive CPU usage that can occur when spilling for window functions.